runtime/runtime.go

```diff
@@ -287,8 +287,8 @@ func newClient(account ProjectEntry) *gsclient.Client {
 		account.Token,
 		false,
 		true,
-		500,
-		0, // no retries
+		1000,
```

Comment:
I think 1 second is not enough to generate kubeconfig in the API/backend. I would recommend 10s.

Reply:
1 second will be problematic if we have two or more processes trying to generate kubeconfig. The backend locks the resource for 8 or more seconds to fulfill the request of the first process the other ones will be timed out. @nvthongswansea

Reply:
We need 10 seconds for the waiting interval between retries.